client/src/charts/wrapped_nivo/NivoLineBarToggle/NivoLineBarToggle.stories.js

```diff
@@ -0,0 +1,53 @@
+import { scaleOrdinal } from "d3-scale";
+import React from "react";
+
+import { newIBCategoryColors } from "src/core/color_schemes";
+
+import { infobase_colors_smart } from "src/charts/wrapped_nivo/wrapped_nivo_common";
+
+import { NivoLineBarToggle } from "./NivoLineBarToggle";
+
+export default {
+  title: "charts/NivoLineBarToggle",
+  component: NivoLineBarToggle,
+};
+
+const Template = (args) => <NivoLineBarToggle {...args} />;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  legend_title: "Legend Title",
+  bar: true,
```

Comment:
I can't help but notice some of these props are obsolete, like `bar` and the options in `graph_options`. And rest of the codebase is still using them who knows for what reasons.. It'd be nice if you could make a separate branch to clean up obsolete props ! (This one can just merge as is I think)

Reply:
A separate branch to clean up obsolete props on this component or the whole infobase?

Reply:
Just this component

Reply:
sounds good 👍